Add more results BEIR-NL (2)

nikolay-banar opened the pull request:
More models for BEIR-NL

<!-- If you are submitting a dataset or a model for the model registry, please use the corresponding checklists below. Otherwise, feel free to remove them. -->

<!-- add additional description, question etc. related to the new dataset -->


## Checklist
<!-- Please do not delete this -->

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the results files checker `make pre-push`.

### Adding a model checklist
<!-- 
When adding a model to the model registry
see also https://github.com/embeddings-benchmark/mteb/blob/main/docs/reproducible_workflow.md
-->

- [x] I have added model implementation to `mteb/models/` [directory](https://github.com/embeddings-benchmark/mteb/tree/main/mteb/models). Instruction to add a model can be found [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/reproducible_workflow.md) in the following PR ____

nikolay-banar committed:
BEIR-NL more results added (2)

KennethEnevoldsen reviewed (changes requested):
Seems like you are missing revisions on all models as well as metadata. That is why the test fails.

Edit: I have also added the checklist back in. Please do not delete it.

nikolay-banar committed:
KaLM v1.5 removed

nikolay-banar commented:
> Seems like you are missing revisions on all models as well as metadata. That is why the test fails.
> 
> Edit: I have also added the checklist back in. Please do not delete it.

I added a wrong model. Sorry for that. It should be fine now.
